### PR TITLE
Fix brute_force_block_structure algorithm check

### PR DESCRIPTION
**Pull request by JohanWBorg**

### CODE REVIEW

File: src/pharmpy/tools/iivsearch/tool.py [hunk 1]
```diff
@@ -155,7 +155,18 @@ def start(
 
     sum_models = [summarize_modelfit_results(input_res)]
 
-    for algorithm_cur in list_of_algorithms:
+    for algorithm_cur in list_of_algorithms.copy():
```
Comment (rikardn) [inline src/pharmpy/tools/iivsearch/tool.py:158]: Do not copy here (see next comment)

File: src/pharmpy/tools/iivsearch/tool.py [hunk 2]
```diff
@@ -155,7 +155,18 @@ def start(
...
+            algorithm_cur == 'brute_force_block_structure'
+            and len(
+                set(base_model_entry.model.random_variables.iiv.names).difference(
+                    _get_fixed_etas(base_model_entry.model)
+                )
+            )
+            <= 1
+        ):
+            list_of_algorithms.remove('brute_force_block_structure')
+            break
```
Comment (rikardn) [inline src/pharmpy/tools/iivsearch/tool.py:169]: Replace with `continue`

**Review by rikardn — CHANGES_REQUESTED** [review]

File: src/pharmpy/tools/iivsearch/tool.py (hunk 2 above)
Comment (JohanWBorg) [reply src/pharmpy/tools/iivsearch/tool.py]: Fixed it by introducing new variable for the applied algorithms

**Review by rikardn — APPROVED** [review]
LGTM!

**Pull request merged**